in/kafkamdm/kafkamdm.go

```diff
@@ -62,108 +77,202 @@ func ConfigProcess(instance string) {
 	if !Enabled {
 		return
 	}
-
-	waitTime, err := time.ParseDuration(consumerMaxWaitTime)
-	if err != nil {
-		log.Fatal(4, "kafka-mdm invalid config, could not parse consumer-max-wait-time: %s", err)
+	switch offset {
+	case "last":
+	case "oldest":
+	case "newest":
+	default:
+		_, err := time.ParseDuration(offset)
```

Comment:
why don't we set a package-level variable here, so that we don't have to re-parse in `Start()`